networkx/readwrite/nx_ete.py

```diff
@@ -0,0 +1,221 @@
+"""
+****
+ETE
+****
+Read and write NetworkX graphs in the New Hampshire Newick format.
+
+"ETE (Environment for Tree Exploration) is a Python programming toolkit
+that assists in the automated manipulation,
+analysis and visualization of phylogenetic trees.
+
+It uses the New Hampshire Newick format as one of the most widely used
+standard representation of trees in bioinformatics."
+
+See http://etetoolkit.org/ for documentation.
+
+Format
+------
+http://etetoolkit.org/docs/latest/tutorial/tutorial_trees.html#reading-and-writing-newick-trees
+
+"""
+
+__all__ = ["to_ete", "from_ete", "write_ete", "read_ete"]
+
+import networkx as nx
+from networkx.utils import open_file, deque
+
+
+def to_ete(G):
+    """Returns a ETE Tree from NetworkX arborescence [3]_ G.
```

Comment:
```suggestion
    """Returns a ETE Tree from NetworkX :mod:`arborescence<tree>` G.
```

The citation syntax should only be used for external references - it's better to use built-in roles like `:ref:` to reference other NetworkX objects.

In fact, we should probably use intersphinx to better integrate with the ETE docs, but let's worry about that in a different PR :)